CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.4)
+
+set(zenfs_SOURCES "fs/fs_zenfs.cc" "fs/zbd_zenfs.cc" "fs/io_zenfs.cc" PARENT_SCOPE)
```

Comment:
I think we should also set the `zenfs_HEADERS` accordingly, so that ZenFS is rebuild when changes in the header files are made.

Reply:
Thanks for pointing that out. zenfs_HEADERS isn't used when building - as I understand it, cmake is able to figure that out on it's own. However, the zenfs_HEADERS is used during install to specify the headers that need to be installed with the package, so we need to specify them.